src/components/KuadrantPoliciesPage.tsx

```diff
@@ -212,6 +213,19 @@ const KuadrantPoliciesPage: React.FC = () => {
     <PoliciesListPage resource={resources[2]} activeNamespace={activeNamespace} />
   );
 
+  const handleNamespaceChange = (activeNamespace: string) => {
+    let currentTab = '';
+    const activeTab = location.pathname.split('/').pop();
```

Comment:
I hit an issue on the 'All Policies' tab after you select an individual namespace more than once.
The url has a duplicate '/policies' in the path
e.g. `http://localhost:9000/kuadrant/ns/kuadrant-system/policies/policies`

It works ok on other tabs (I believe because of the activeTab hanging off the `/policies` like `/policies/dns`,
but not on the 'All Policies' tab because the code pops off the end of the path to determine the activeTab